source/extensions/filters/http/jwt_authn/token_cache.cc

```diff
@@ -0,0 +1,31 @@
+#include "extensions/filters/http/jwt_authn/token_cache.h"
+
+using std::chrono::system_clock;
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+TokenCache::TokenCache(int cache_size)
+    : SimpleLRUCache<std::string, ::google::jwt_verify::Jwt>(cache_size) {}
+
+TokenCache::~TokenCache() { clear(); }
+
+::google::jwt_verify::Jwt* TokenCache::find(const std::string& token) {
+  ::google::jwt_verify::Jwt* jwt_cache;
+  TokenCache::ScopedLookup lookup(this, token);
+  if (lookup.found()) {
```

Comment:
I like to just cached jwt directly.  We should not use ScopedLookup,  we need to call Release() the key after authenticator is done.

Reply:
Done.